back-end/legeno-around-here/src/test/java/wooteco/team/ittabi/legenoaroundhere/domain/area/RegionLevelTest.java

```diff
@@ -0,0 +1,32 @@
+package wooteco.team.ittabi.legenoaroundhere.domain.area;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class RegionLevelTest {
+
+    @DisplayName("정렬 학습")
+    @Test
+    void sort() {
+        //given
+        List<RegionLevel> regionLevels = Arrays.asList(RegionLevel.LEVEL_1,
```

Comment:
`LEVEL_1`을 아래로 내리면 가독성이 더 좋아질 것 같아요!! 
```suggestion
        List<RegionLevel> regionLevels = Arrays.asList(
        RegionLevel.LEVEL_1,
```

Reply:
넵